lib/cdo/user_helpers.rb

```diff
@@ -20,7 +20,7 @@ def self.generate_username(queryable, name)
     return prefix if queryable.where(username: prefix).limit(1).empty?
 
     # Throw darts to find an appropriate suffix, using it if we hit bullseye.
-    (0..1).each do |exponent|
+    (0..2).each do |exponent|
       min_index = 10**exponent
       max_index = 10**(exponent + 1) - 1
       3.times do |_i|
```

Comment:
The average `LIKE` table scan query takes approximately 300ms. The average indexed lookup takes approximately 20ms. Though I'm happy to increase the number of darts further (next, I'd probably change the `3.times` to `5.times`), I think it makes sense to do so in steps.